sematic/ui/src/types/Types.tsx

```diff
@@ -70,6 +70,13 @@ export type TypeSerialization = {
   registry: TypeRegistry;
 };
 
+export type TypeGitInfo = {
+  remote: string;
+  branch: string;
+  commit: string;
+  dirty: boolean;
+}
```

Comment:
This should be in `Models.tsx`. `Types.tsx` is for artifact visualization.

Reply:
Done.